### PR TITLE
Adding quoter flashBull tests

### PR DESCRIPTION
**Pull request by alpinechicken**
# Task:

## Description

- Fixes test calculation for total eth
- Adds quota

Fixes # (issue)

## Type of change

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings
- [x ] Added video recordings if it is a UI change

## User Facing Checklist

- [x ] I fully understand the user problem this PR is solving
- [x ] I know who the target user is for this PR and have a deep understanding of that user
- [ x] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol [hunk 1]
```diff
@@ -12,6 +12,8 @@ import { IEulerMarkets } from "../../src/interface/IEulerMarkets.sol";
 import { IEulerEToken } from "../../src/interface/IEulerEToken.sol";
 import { IEulerDToken } from "../../src/interface/IEulerDToken.sol";
 // contract
+import { SwapRouter } from "v3-periphery/SwapRouter.sol";
```
Comment (aleone) [inline packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol:15]: do we need the swap router?

File: packages/bull-vault/test/integration-test/FlashBullTestFork.t.sol [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { IEulerMarkets } from "../../src/interface/IEulerMarkets.sol";
 import { IEulerEToken } from "../../src/interface/IEulerEToken.sol";
 import { IEulerDToken } from "../../src/interface/IEulerDToken.sol";
 // contract
+import { SwapRouter } from "v3-periphery/SwapRouter.sol";
```
Comment (aleone) [inline packages/bull-vault/test/integration-test/FlashBullTestFork.t.sol:14]: dont need swap router right?

**Review by aleone — APPROVED** [review]
lgtm

File: packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol (hunk 1 above)
Comment (alpinechicken) [reply packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol:15]: no

**Pull request merged**